nerfstudio/data/dataparsers/colmap_dataparser.py

```diff
@@ -392,13 +395,49 @@ def _generate_dataparser_outputs(self, split: str = "train", **kwargs):
         return dataparser_outputs
 
     def _load_3D_points(self, colmap_path: Path, transform_matrix: torch.Tensor, scale_factor: float):
+        points3D = None
+        if (colmap_path / "scene.obj").exists():
+            mesh_scene = trimesh.load(colmap_path / "scene.obj", force='mesh')
+            vertices = mesh_scene.vertices # (num_pt, 3)
+            faces = mesh_scene.faces # (num_faces, 3)
+            triangles = torch.tensor(mesh_scene.triangles).float()  # equal vertices[faces]
+            # rotated y-up world frame to z-up world frame
+            vertices = np.einsum('ij,kj->ki', rotx(np.pi / 2), vertices)
+            vertices = torch.from_numpy(vertices.astype(np.float32))
+
+            num_pts_each_triangle = 1 #self.config.num_splats
```

Comment:
Is this line of code meant to be hard-coded?

Reply:
solve in 3ef7892a3a4efafe6267d2f4acf8247b62a60886